Update (#4)
DevExpress-Examples/blazor-dxgrid-disable-selection-checkboxes-by-condition: README.md

[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Grid for Blazor - Disable Selection Checkboxes in Specific Rows

> This example illustrates how to prevent users from selecting specific data items in the [DevExpress Grid for Blazor](https://docs.devexpress.com/Blazor/403143/grid). In this example, the Grid component disables the selection column's checkboxes for items, whose `Summary` field is set to `Mild`.

Reviewer: > the Grid component disables **the selection column's checkboxes for items**, whose `Summary` field is set to `Mild`

1. This phrase looks strange, I cannot say I understand it properly...
2. You don't need a comma before the word "whose". Please don't forget that punctuation works differently in English.
3. The word "whose" points at "items" and adds grammatical definiteness to them, therefore, a definite article becomes necessary.

Did you intend to say the following?

```suggestion
This example illustrates how to prevent users from selecting specific data items in the [DevExpress Grid for Blazor](https://docs.devexpress.com/Blazor/403143/grid). In this example, the Grid component disables column selection checkboxes for the items whose `Summary` field is set to `Mild`.
```

Author: Not really. I intended to say the following:
```suggestion
This example illustrates how to prevent users from selecting specific data items in the [DevExpress Grid for Blazor](https://docs.devexpress.com/Blazor/403143/grid). In this example, the Grid component displays the [selection column](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSelectionColumn). This column contains checkboxes that allow users to select and deselect grid items. The component disables the checkboxes for the items whose `Summary` field is set to `Mild`.
```
or
```suggestion
This example illustrates how to prevent users from selecting specific data items in the [DevExpress Grid for Blazor](https://docs.devexpress.com/Blazor/403143/grid). The Grid component's [selection column](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSelectionColumn) displays checkboxes that allow users to select and deselect items. In this example, the component disables these checkboxes for the items whose `Summary` field is set to `Mild`.
```

Reviewer: Now it makes sense, thank you. I like the second option.

> This example illustrates how to prevent users from selecting specific data items in the [DevExpress Grid for Blazor](https://docs.devexpress.com/Blazor/403143/grid). The Grid component's [selection column](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSelectionColumn) displays checkboxes that allow users to select and deselect items. In this example, the component disables these checkboxes for the items whose `Summary` field is set to `Mild`.

Reviewer 2: ```suggestion
This example illustrates how to prevent users from selecting specific data items in the [DevExpress Grid for Blazor](https://docs.devexpress.com/Blazor/403143/grid). The Grid component's [selection column](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridSelectionColumn) displays checkboxes that allow users to select and deselect items. In this example, the component disables these checkboxes for items whose `Summary` field is set to `Mild`.
```
